content/en/news/releases/1.7.x/announcing-1.7/upgrade-notes/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.7.
+weight: 20
+---
+
+When you upgrade from Istio 1.6.x to Istio 1.7.x, you need to consider the changes on this page.
+These notes detail the changes which purposefully break backwards compatibility with Istio 1.6.x.
+The notes also mention changes which preserve backwards compatibility while introducing new behavior.
+Changes are only included if the new behavior would be unexpected to a user of Istio 1.6.x.
+
+## Require Kubernetes 1.16+
+Kubernetes 1.16+ is now required for installation.
```

Comment:
Do we really only have one breaking change for 1.7? That's all that made it in the release notes, but usually we have quite a few.

Reply:
I would highlight the following breaking changes for the users, from our release notes :)

Reply:
- `istioctl manifest apply` is removed, please switch to use `istioctl install` instead.

Reply:
installation of telemetry addons by `istioctl` is deprecated, please use these [addons integration](/docs/ops/integrations/)  instructions.

Reply:
What is our statement around mixer for 1.7?  Should not we mention anything?  Not shipped by default and no longer supported in Istio?

cc @mandarjog @douglas-reid

Reply:
@linsun I've added both suggestions, thanks.

Reply:
@linsun 1.7 is the last release in which Mixer is supported. I'm not sure that anything has changed in status for this release. It definitely needs to be called out in 1.8, however.

Reply:
Thanks @douglas-reid !